### PR TITLE
JBPM-5074 Process variable text overflows tooltip

### PR DESCRIPTION
**Pull request by bxf12315**
@cristianonicolai 
Please review it.

### CODE REVIEW

**Comment by ederign** [comment]
@cristianonicolai please assign this for someone for qe :)

**Comment by cristianonicolai** [comment]
Jenkins execute full downstream build

File: uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/tables/PopoverTextCell.java [hunk 1]
```diff
@@ -37,6 +39,8 @@
  */
 public class PopoverTextCell extends AbstractSafeHtmlCell<String> {
 
+    protected static final CommonCss css = CommonResources.INSTANCE.CSS();
```
Comment (jstastny-cz) [inline uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/tables/PopoverTextCell.java]: constant naming should be upper case and words divided by underscore:
css -> CSS

File: uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/main/resources/org/uberfire/ext/widgets/common/client/resources/css/common.css [hunk 2]
```diff
@@ -219,4 +219,8 @@
 
 .btn-file > span + i {
     padding-left: 2px;
-}
\ No newline at end of file
+}
+
+.popoverTextBreakWord{
```
Comment (cristianonicolai) [inline uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/main/resources/org/uberfire/ext/widgets/common/client/resources/css/common.css]:
```suggestion
.popoverTextBreakWord {
```

Comment (bxf12315) [reply uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/main/resources/org/uberfire/ext/widgets/common/client/resources/css/common.css]: Done

File: uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/tables/PopoverTextCell.java (hunk 1 above)
Comment (bxf12315) [reply uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/tables/PopoverTextCell.java]: Done

**Comment by cristianonicolai** [comment]
Jenkins execute full downstream build

**Comment by ederign** [comment]
Closing due to inactivity. @bxf12315  feel free to reopen it if necessary.

**Pull request closed without merging**